roles/aodh/templates/etc/serverspec/aodh_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
+
+[['aodh.conf', 640], ['api_paste.ini', 640], ['policy.json', 644]].each do |file|
+  describe file("/etc/aodh/#{file[0]}") do
+    it { should be_mode file[1] }
+    it { should be_owned_by 'aodh' }
+    it { should be_grouped_into 'aodh' }
+  end
+end
+
+describe file('/var/log/aodh/aodh-api.log') do
+  it { should be_mode 644 }
+  it { should be_owned_by 'aodh' }
+end
+
+describe file('/var/log/aodh/aodh-evaluator.log') do
+  it { should be_mode 644 }
+  it { should be_owned_by 'aodh' }
+end
+
+describe file('/var/log/aodh/aodh-expirer.log') do
+  it { should be_mode 644 }
+  it { should be_owned_by 'aodh' }
+end
+
+describe file('/var/log/aodh/aodh-listener.log') do
+  it { should be_mode 644 }
+  it { should be_owned_by 'aodh' }
+end
+
+describe file('/var/log/aodh/aodh-notifier.log') do
+  it { should be_mode 644 }
+  it { should be_owned_by 'aodh' }
+end
+
+describe file('/etc/logrotate.d/aodh') do
+  it { should exist }
+  file_contents = [ '# Generated by Ansible.',
+  		     '# Local modifications will be overwritten.',
+		     '',
+		     '/var/log/aodh/*.log',
+		     '{',
+		     '  daily',
+		     '  missingok',
+		     '  rotate 7',
+		     '  compress',
+		     '}'].join("\n")
+
+  it { should contain file_contents }
+end
+
+describe file('/etc/aodh/aodh.conf') do
+  it { should contain 'debug = {{ aodh.debug }}' }
+  it { should contain 'log_dir = /var/log/aodh' }
+  it { should contain 'os_auth_url = {{ endpoints.keystone.url.admin }}' }
+end
```

Comment:
Do we have tech spec entries for all these things we're looking for? I think we're supposed to reference line items somehow.